src/data/markdown/docs/01 guides/02 Using k6/14 Scenarios/01 Executors.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Executors'
+---
+
+> ### 🎉 New in v0.27.0
```

Comment:
Feedback -> Besides adding this `note` to the `Scenarios` main page, I am not sure about adding it in all the other pages.